crates/payload/primitives/src/lib.rs

```diff
@@ -33,7 +33,16 @@ mod payload;
 pub use payload::{ExecutionPayload, PayloadOrAttributes};
 
 /// The types that are used by the engine API.
-pub trait PayloadTypes: Send + Sync + Unpin + core::fmt::Debug + Clone + 'static {
+pub trait PayloadTypes:
+    Send
+    + Sync
+    + Unpin
+    + core::fmt::Debug
+    + Clone
+    + serde::Serialize
+    + for<'de> serde::Deserialize<'de>
```

Comment:
is moving these bounds required here?
or could they remain on EngineTypes?

Reply:
good point, not required at all, moved back here